CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+# Cyberpunk 2077 Vortex Support v0.10.0-a "River"
+
+> [Insert good copy that is clever, engaging, and better than something a simple developer can come up with]
+
+Update directly in Vortex, or get the release on the Nexus page (or Github): https://www.nexusmods.com/site/mods/196?tab=files
+
+Why use a mod manager? In addition to supporting REDmods and every other kind of mod there is for CP2077, there's the quality-of-life improvements like automatically fixing many common types of mod structure problems that you'd have to do by hand, clear (?) instructions on any possile additional steps for any given mod or problem encountered, and the easy ability to enable/disable mods, you also get profiles to manage separate sets of mods more easily, update notifications, much easier game upgrades, easy toggling between multiple versions or variants (e.g. colors) of a mod, Collections, and all sorts of nice things. And you can of course install mods from any source - even the mod showcase channel, and they do not even need to be in a zipfile. We gotchu.
+
+As always, all feedback is appreciated so we can ensure that no bugs or oversights have slipped through our testing — and any suggestions and ideas are welcome too! Please get in touch either on Discord, Github, or Nexus (links at the bottom).
+
+## Main Jobs
+
+- ✨ **Modlist File Support** The latest cyberpunk update added a feature to redmod to allow deploying mods specified in a file. When we read that, we knew we had to get right on it since many people have been running into a limit on mods because of how long the deploy command is. This will fix that right up.
```

Comment:
Highlight the user-facing “No more mod count limit on REDmods (Eat it NetWatch!)” or so?

Reply:
I like that. Any ideas for the release copy? It's fine if not.